FunctionalTableData/TableView/FunctionalTableData.swift

```diff
@@ -283,6 +283,10 @@ public class FunctionalTableData: NSObject {
 			
 			strongSelf.doRenderAndDiff(newSections, animated: animated, animations: animations, completion: completion)
 		}
+		renderAndDiffQueue.operations.forEach { (operation) in
```

Comment:
`renderAndDiffQueue.operations.lazy.filter { !$0.isExecuting }.forEach { $0.cancel() }`

Reply:
Won’t this iterate over the array twice instead of once? Why lazy?

Reply:
lazy makes it iterate over elements only once :)
```
["A", "B", "CD", "EF"].lazy.filter { 
    print("filter") 
    return $0.count > 1 
}.forEach { 
    print("forEach") 
    $0.lowercased() 
}
```
spits out:
```
filter
filter
filter
forEach
filter
forEach
```

Reply:
(but I don't actually care, lazy is pretty cool though and makes for nice functional code without losing efficiency)

Reply:
TIL.gif